src/tests/suite/colorize-results/string-tagname_js.json

```diff
@@ -109,7 +109,7 @@
 	},
 	{
 		"c": "div",
-		"t": "source.js meta.var.expr.js string.quoted.double.ts",
+		"t": "source.js meta.var.expr.js string.quoted.double.js.jsx",
```

Comment:
How come the extensions have changed to jsx.js here?

Reply:
Yeah that's the part I'm trying to fix if possible. `source.ts#expression` doesn't parse JSX so I added `source.js.jsx` as one of the `includes`. But that changed some of these scopes in unwanted places. Although if you open VSCode you don't really see the difference in colors.